pkg/controller/plan/kubevirt.go

```diff
@@ -1644,7 +1650,7 @@ func vmOwnerReference(vm *cnv.VirtualMachine) (ref meta.OwnerReference) {
 
 // TODO move elsewhere
 func (r *KubeVirt) createOpenStackVolumes(vm ref.Ref) (err error) {
-	secret, err := r.ensureSecret(vm, false)
+	secret, err := r.ensureSecret(vm, r.secretDataSetterForCDI(vm))
```

Comment:
for openstack it doesn't make much sense as you clone the data and we don't use CDI.. I'll change that